staking/src/lib.rs

```diff
@@ -0,0 +1,240 @@
+#![no_std]
+use codec::{Decode, Encode};
+use gstd::{exec, msg, prelude::*, ActorId};
+use scale_info::TypeInfo;
+use staking_io::*;
+pub mod ft_messages;
```

Comment:
Do we really need a dedicated module for one function?

Reply:
We discussed this function(`transfer_tokens()`) in Telegram. Returned it to the `lib.rs` module, but outside the `impl Staking`